apps/codesandbox-react-next-template/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "codesandbox-react-next-template",
+  "description": "Template for creating codesandbox environment for @fluentui/react-next packages.",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "just-scripts build",
+    "lint": "just-scripts lint",
+    "just": "just-scripts",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.0",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@fluentui/scripts": "^1.0.0",
+    "typescript": "3.7.2"
+  },
+  "dependencies": {
+    "@microsoft/load-themed-styles": "^1.10.26",
+    "@fluentui/react-next": "^9.0.0-alpha.26",
```

Comment:
`react-next` was removed in https://github.com/microsoft/fluentui/pull/16239. Do we need this?

Reply:
this is from a bad merge i think .. looking into resolving and reverting

Reply:
redid the merge and should be up to date